cli_reference/differences_oc_kubectl.adoc

```diff
@@ -0,0 +1,60 @@
+[[cli-reference-differences-oc-kubectl]]
+= Differences Between oc and kubectl
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[differences-oc-kubectl-why-use]]
+== Why Use oc Over kubectl?
+
+Kubernetes' command line interface (CLI), `kubectl`, is used to run commands
+against any Kubernetes cluster. Because {product-title} runs on top of a
+Kubernetes cluster, a copy of `kubectl` is also included with `oc`,
```

Comment:
> OpenShift Container Platform runs on top of a Kubernetes cluster

The PR LGTM except the statement here. This appears to show in OCP cluster there is a Kubernetes cluster that exists.